idutils/config.py

```diff
@@ -0,0 +1,116 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of IDUtils
+# Copyright (C) 2024 CERN.
+#
+# IDUtils is free software; you can redistribute it and/or modify
+# it under the terms of the Revised BSD License; see LICENSE file for
+# more details.
+#
+# In applying this license, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+"""Default configuration."""
+
+from .utils import *
+
+PID_SCHEMES = [
+    ("doi", is_doi),
+    ("ark", is_ark),
+    ("handle", is_handle),
+    ("purl", is_purl),
+    ("lsid", is_lsid),
+    ("urn", is_urn),
+    ("ads", is_ads),
+    ("arxiv", is_arxiv),
+    ("ascl", is_ascl),
+    ("hal", is_hal),
+    ("pmcid", is_pmcid),
+    ("isbn", is_isbn),
+    ("issn", is_issn),
+    ("orcid", is_orcid),
+    ("isni", is_isni),
+    ("ean13", is_ean13),
+    ("ean8", is_ean8),
+    ("istc", is_istc),
+    ("gnd", is_gnd),
+    ("ror", is_ror),
+    ("pmid", is_pmid),
+    ("url", is_url),
+    ("sra", is_sra),
+    ("bioproject", is_bioproject),
+    ("biosample", is_biosample),
+    ("ensembl", is_ensembl),
+    ("uniprot", is_uniprot),
+    ("refseq", is_refseq),
+    ("genome", is_genome),
+    ("geo", is_geo),
+    ("arrayexpress_array", is_arrayexpress_array),
+    ("arrayexpress_experiment", is_arrayexpress_experiment),
+    ("swh", is_swh),
+    ("viaf", is_viaf),
+]
+"""Definition of scheme name and associated test function.
+
+Order of list is important, as identifier scheme detection will test in the
+order given by this list."""
+
+IDUTILS_EXTENDED_PID_SCHEMES = []
```

Comment:
I would have a preference for `CUSTOM` instead, to clearly make that they are your own ones.
And also... shall we enable extension via entry points instead?